package.json

```diff
@@ -26,6 +26,7 @@
     "typescript": "^5.1.6"
   },
   "resolutions": {
-    "dompurify": "3.1.3"
+    "dompurify": "3.1.3",
+    "cross-spawn": "7.0.5"
```

Comment:
should we be using the ^ here so we can still use future versions that are compatible with 7.0.5? `^7.0.5`

Reply:
Yea that makes sense to me! I've updated to include this.